lib/column/decimal.go

```diff
@@ -6,7 +6,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/juju/errors"
+	"errors"
```

Comment:
I think built-in package "errors" import statement should be adjacent with other built-in package imports, with no blank line.

Reply:
update format